### PR TITLE
Add host name env variable

### PR DESCRIPTION
**Pull request by gparant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,17 @@ triggered when you move next to a colleague).
...
 
-Install Docker.
+Install Docker : https://docs.docker.com/get-docker/
+Install docker-compose : https://docs.docker.com/compose/install/
+
+**Add your local environment variable**
+Run:
+
+```
+cp .en.template .env
```
Comment (djmaze) [inline README.md:27]: `.env.template`

File: README.md [hunk 2]
```diff
@@ -17,7 +17,17 @@ triggered when you move next to a colleague).
...
+```
+_``If you want deploy on the dev server, you must update HOST_NAME in your `.env` by your private domain (workadventure.localhost => yourdomain.com).``
```
Comment (djmaze) [inline README.md:29]: - `If you want to deploy on the dev server, you must update HOST_NAME in your `.env` with your private domain`

File: README.md [hunk 3]
```diff
@@ -17,7 +17,17 @@ triggered when you move next to a colleague).
...
+Install docker-compose : https://docs.docker.com/compose/install/
+
+**Add your local environment variable**
+Run:
+
+```
+cp .en.template .env
+```
+_``If you want deploy on the dev server, you must update HOST_NAME in your `.env` by your private domain (workadventure.localhost => yourdomain.com).`` 
+**_``Don't forgot to add A entry in DNS like this *.yourdomain.com. The different deployed for WorkAdventure are: play. ; pusher. ; maps. ; api. ; uploader. ;``_**
```
Comment (djmaze) [inline README.md:30]: `Don't forgot to add A entries in DNS like this: *.yourdomain.com. The different subdomains deployed for WorkAdventure are:`

**Review by djmaze — CHANGES_REQUESTED** [review]
I corrected some mistakes in the README. Otherwise, looks good to me! (But I am not a contributor.)

File: README.md (hunk 3 above)
Comment (jalr) [reply README.md:30]: Don't forget

**Comment by jalr** [comment]
nice improvement :+1:

**Comment by Informatic** [comment]
Tip: Instead of relying on people manually copying .env template you should be able to use following syntax for default values:
`${HOST_NAME:-workadventure.localhost}`

**Comment by gparant** [comment]
> Tip: Instead of relying on people manually copying .env template you should be able to use following syntax for default values:
> `${HOST_NAME:-workadventure.localhost}`

It's a great idea I think that it will be more easy for any dev. But we have some configurations in `.env` file and the idea is to deploy in any server easily. If you never copy `.env.template` file, you couldn't manage the JITSI meet configuration and apply your host name.

So if the application doesn't work, I hope that the dev could check the redme and copy this file 🤞 😄

**Comment by moufmouf** [comment]
Honest question: is this a real improvement?

The `docker-compose.yaml` is only targetted at development environments. It should not be used in production (for instance, it is building JS files in dev mode that weight around 13MB and enables live reload).

So do we really want to offer the possibility for a developer to change the URL on his local environment?
I don't want to give the false feeling that this can be used in production (especially since we have a number of other PRs that are proposing to add a separate docker-compose for prod)

**Comment by moufmouf** [comment]
Closing. The `docker-compose.yaml` is only targetted at development environments.
There are other open PRs with a `docker-compose-prod.yaml` dedicated to production environment that are better suited.

**Pull request closed without merging**